REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+# Reindexing
+
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
+
+## How to run a reindex
+
+Running a reindex will require:
+
+- Preparing and deploying a release
+- Terraforming a new pipeline using that release
+- Running the reindex script to feed the new pipeline
+
+### Update the catalogue-pipeline environments
+
+The catalogue-pipeline is deployed using [weco-deploy](https://github.com/wellcomecollection/weco-deploy) but does not deploy in the usual way as there is often one release per pipeline. 
+
+Each environment will represent a new release.
+
+In `.wellcome_project`:
+
+```
+catalogue_pipeline:
+  environments:
+    - id: "2021-06-27"
+      name: "2021-06-27"
+    - id: "2021-07-06"
+      name: "2021-07-06"
+    - id: "MY-NEW-PIPELINE"
+      name: "MY-NEW-PIPELINE"
+```
```

Comment:
Suggestion:

<pre>
```diff
 catalogue_pipeline:
   environments:
     - id: "2021-06-27"
       name: "2021-06-27"
     - id: "2021-07-06"
       name: "2021-07-06"
+    - id: "YYYY-MM-DD"
+      name: "YYYY-MM-DD"
```
</pre>

which renders as:

```diff
 catalogue_pipeline:
   environments:
     - id: "2021-06-27"
       name: "2021-06-27"
     - id: "2021-07-06"
       name: "2021-07-06"
+    - id: "YYYY-MM-DD"
+      name: "YYYY-MM-DD"
```

to be super clear about what somebody should be adding.